src/Courses/TopicGrades/GradeInfoHeader.tsx

```diff
@@ -0,0 +1,247 @@
+import React, { useState, useEffect, useRef } from 'react';
+import _ from 'lodash';
+import { ProblemObject, StudentGrade, StudentGradeDict, StudentWorkbookInterface, UserObject } from '../CourseInterfaces';
+import logger from '../../Utilities/Logger';
+import { Button, FormControl, Grid, InputLabel, makeStyles, Modal, Select, TextField } from '@material-ui/core';
+import { OverrideGradeModal } from '../CourseDetailsTabs/OverrideGradeModal';
+
+enum OverrideGradePhase {
+    PROMPT = 'PROMPT',
+    CONFIRM = 'CONFIRM',
+    LOCK = 'LOCK',
+    LOCK_CONFIRM = 'LOCK_CONFIRM',
+}
+interface GradeInfoHeaderProps {
+    grade: StudentGradeDict;
+    workbookId?: number;
+    selected: {
+        problem?: ProblemObject,
+        user?: UserObject,
+        workbook?: StudentWorkbookInterface,
+    };
+    setSelected: React.Dispatch<React.SetStateAction<{
+        problem?: ProblemObject,
+        user?: UserObject,
+        workbook?: StudentWorkbookInterface,
+    }>>;
+    onSuccess: React.Dispatch<React.SetStateAction<{
+        id?: number, 
+        override?: Partial<StudentGrade>
+    }>>;
+}
+
+type WorkbookOption = {
+    value: number;
+    label: string;
+};
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Woah, is this built-in to React? I don't see an import.

Reply:
Oh, nevermind, I see it in material-core. Cool! I'm glad we're doign something on styling.

Reply:
makeStyles is from material-ui/core